### PR TITLE
Tables added

### PR DESCRIPTION
**Pull request by IshanMi**
Results now appear in tables, hyperlinks connect all the pages.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@ __pycache__
 **.pyc
 venv
 src/reference_patents/
+.env.example
```
Comment (bbelderbos) [inline .gitignore:5]: This one you don't want to ignore, it is the file I would copy to set up my env

File: src/templates/results.html [hunk 2]
```diff
@@ -10,22 +10,27 @@ <h1 style="text-align: center"> Reverse Image Search for Patents! </h1>
...
-      <span class="ID"> <h3> USPTO #{{ applications[i] }} </h3></span>
-  </div>
-  <div>
-      <span class="image"> <img src="../static/images/{{ images[i]}}" alt="Patent Art" width="400" height="500"> </span>
-  </div>
-  </span>
+      {% if i % 2 == 0 %}
+          <tr>
+          <th> USPTO #{{ applications[i] }}</th>
+          <td> <img src="../static/images/{{ images[i] }}" alt="Patent Art" width="400" height="500"></td>
```
Comment (bbelderbos) [inline src/templates/results.html:26]: Use `{% static %}`

File: src/templates/results.html [hunk 3]
```diff
@@ -10,22 +10,27 @@ <h1 style="text-align: center"> Reverse Image Search for Patents! </h1>
...
+      {% else %}
+          <th> USPTO #{{ applications[i] }}</th>
+          <td> <img src="../static/images/{{ images[i] }}" alt="Patent Art" width="400" height="500"></td>
```
Comment (bbelderbos) [inline src/templates/results.html:29]: ditto

File: src/templates/results.html (hunk 2 above)
Comment (IshanMi) [reply src/templates/results.html:26]: Isn't {% %} used for control statements like if-else statements?

Did you mean using {{ static }} and  render_template("results.html", static=app.static_folder) ? Or did I not understand

Comment (bbelderbos) [reply src/templates/results.html:26]: Sorry my memory failed me, seems you can actually use `url_for` here too: https://flask.palletsprojects.com/en/2.0.x/tutorial/static/

**Pull request merged**